src/Paramore.Brighter.Outbox.Sqlite/SqliteOutboxSync.cs

```diff
@@ -41,11 +41,8 @@ namespace Paramore.Brighter.Outbox.Sqlite
     /// </summary>
     public class SqliteOutboxSync : RelationDatabaseOutboxSync<SqliteConnection, SqliteCommand, SqliteDataReader, SqliteParameter>
     {
-        private static readonly ILogger s_logger = ApplicationLogging.CreateLogger<SqliteOutboxSync>();
```

Comment:
Again, this forms a standard pattern for us and is likely to be needed - so let's leave it